calculator.py

```diff
@@ -1,11 +1,21 @@
 def addition(number_1, number_2):
-
+    return number_1+number_2
 def subtraction(number_1, number_2):
-
+    return number_1-number_2
 def multiplication(number_1, number_2):
-
+    return number_1*number_2
 def division(number_1, number_2):
-
+    return number_1/number_2
 def get_input():
-
-input()
\ No newline at end of file
+    num1, operate, num2 = input().split()
+    num1=int(num1)
+    num2=int(num2)
+    if(operate=='+'):
+        print(addition(num1,num2))
+    if(operate=='-'):
```

Comment:
Why are you using if 4 times? Use **if else if** statement.
if statement will check all 4 conditions.